Sources/ScipioKit/PackageKind+enabledKinds.swift

```diff
@@ -0,0 +1,7 @@
+import PackageManifestKit
+
+extension Target.TargetKind {
+    static var enabledKinds: [Self] {
```

Comment:
It's difficult to understand what `enabled` means.

It's better to be function of `FrameworkProducer` rather than the extension.

`func shouldBuild(_ target: Target.TargetKind)`

Reply:
`enabledKinds` is also used in `PackageResolver`, so I prefer not to include it within `FrameworkProducer`.